styles.css

```
@@ -0,0 +1,25 @@
+:root{
+    --primary-colour:blue;
+    --secondary-coluor:#44686e;
+}
+
+img {
+    width: 100 display: block height: 70px;
```

I would kindly advise you to use a semi-colon after the properties of the width. and not a full colon, This is advised because you could have multiple attributes separated with a space. and it also indicates you have terminated your statement. (It is CSS best practises)

```suggestion
    width: 100 display;
    block height: 70px;
```

---

Please, let all your comments include an explanation, a "why", for each requested change. This will help them see the reasons with you. Some reasons might seem obvious to you but not to the author of the code.